src/component/header.tsx

```diff
@@ -0,0 +1,40 @@
+
+import { Box, Flex, Link, Image } from '@chakra-ui/react';
+
+
+
+export const Header = () => {
+
+  return (
+    <Box   p='24px' paddingLeft='80px'>
+    <Flex align="center" >
+      
+      <Link href="/" _hover={{ textDecoration: 'none' }}>
+        <Image src="/src/assets/logoImg.svg" alt="Logo" width="100px" height="48px" />
+      </Link>
+   
+      <Box paddingLeft='64px'>
+      <Flex gap='32px'>
+        <Link href="/" fontSize="20px"  _hover={{ textDecoration: 'underline', color: '#F47327', fontWeight: '500' }}>
```

Comment:
```suggestion
        <Link href="/" fontSize="20px"  _hover={{ color: '#F47327', fontWeight: '500' }}>
```
Só remover o textDecoration